### PR TITLE
58 searchbar component universal

### PR DESCRIPTION
**Pull request by OmarKarame**

### CODE REVIEW

**Review by AdrianRotila — CHANGES_REQUESTED** [review]
Searchbar tocamel case

File: src/components/SearchBar/SearchBar.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import "./SearchBar.scss";
+
+import magnifyGlass from "../../assets/magnifying-glass-grey.svg";
+
+const SearchBar = ({placeholderText, searchWord, handleSearchInput }) => {
```
Comment (AdrianRotila) [inline src/components/SearchBar/SearchBar.jsx:5]: placeholder to placeHolder

File: src/components/SearchBar/SearchBar.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    border-radius: 3.31rem;
+    border: 0.063rem;
+    border-color: $color-white;
+    &__input {
+        height: 0.94rem;
+        width: 17.12rem;
+        color: $color-grey-light;
+        line-height: 0.97rem;
+        border: none;
+        //These will need to be changed to new variables
```
Comment (AdrianRotila) [inline src/components/SearchBar/SearchBar.scss:20]: Please remove these comments

File: src/components/SearchBar/SearchBar.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  
+  return (
+    <div className="search-bar">
+      {/*Props being used to change the placeholder text, as well as store search word and handle the logic of the search bar for updates*/}
+      <img src={magnifyGlass} alt="Search Icon" className="search-bar__image"/>
+      <input type="text" placeholder={placeholderText} value={searchWord} onChange={handleSearchInput} className="search-bar__input" data-testid="search-input"/>
```
Comment (AdrianRotila) [inline src/components/SearchBar/SearchBar.jsx:11]: data-testid?

**Review by AdrianRotila — CHANGES_REQUESTED** [review]

**Review by AdrianRotila — APPROVED** [review]

**Review by JosieAilsa — APPROVED** [review]

**Pull request merged**